Bio/SeqIO/NibIO.py

```diff
@@ -138,15 +144,8 @@ def __init__(self, handle):
             raise RuntimeError("unexpected system byte order %s" % byteorder)
         handle.write(bytes.fromhex(signature))
 
-    def write_file(self, records):
+    def write_record(self, record):
         """Use this to write an entire file containing the given record."""
```

Comment:
The comment seems a little confusing given you are using ``.write_record()`` now.

Reply:
OK